cantherm/statmech/kinetics.py

```diff
@@ -0,0 +1,71 @@
+class Reaction:
+    '''
+    Attributes:
+
+        reactants ():
+
+        ts ():
+
+        temp ():
+
+        reac_type ():
+
+        products ():
+
+        tunneling ():
+
+        scale ():
+
+        rates ():
+
+        activation_energy ():
+
+    '''
```

Comment:
For these let's switch over to the NumPy style for documentation. If you're using VSCode you can use [this](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) extension to generate the templates for your docstrings. Otherwise you can follow the style in `cantherm/statmech` or `cantherm/chemistry`.